css/styles.css

```diff
@@ -0,0 +1,63 @@
+#down-button {
+  margin: 3.5em auto;
+  border: solid 0.1px;
+  padding: 0.2em;
+  border-radius: 10px;
+  border-color: gray;
+}
+
+.background {
+  margin: 0;
+  height: 600px;
+  background-image: url("../background.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.list-format {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-around;
+  margin: auto;
+  padding: 0;
+  width: 30%;
+}
+
+.font-color {
+  color: #172b4d;
+}
```

Comment:
- kindly add a line-height: 24px to the body or to this class to improve the spacing in your text to improve your design.